torch/csrc/jit/passes/quantization.cpp

```diff
@@ -725,6 +754,10 @@ graph(%self, %input, %inplace):
           add_functional_relu,
           bn_relu,
           bn_functional_relu,
+          mul_module_relu,
+          inplace_mul_module_relu,
+          mul_functional_relu,
+          inplace_mul_functional_relu,
```

Comment:
I saw we only have two entries for `add_relu` right now, could you add another PR to add two more entries for `add_relu` as well?